### DISCUSSION TITLE
How to invite new users and send signup email from a form in app

### DISCUSSION THREAD

**Discussion by kaberlock** [Budibase/budibase#13486] [Help]
I want to be able to input data about a new user when inviting users from a screen in my app. When I submit the form I want to be able to trigger the invitation email so that manager can create their login credentials etc.

The user will be a manager and invite users to the app and use the app differently and, so I will use this same method for their downline to create their login.

Thank you so much in advance.

**Comment by ConorWebb96** [comment]
Hey @kaberlock,

In this case you'd likely have to use our [public API](https://docs.budibase.com/reference/usercreate). This + the [rest data source](https://docs.budibase.com/docs/rest) is what you would need in order to do this. This discussion here should help you with this. https://github.com/Budibase/budibase/discussions/6098

**Reply by kaberlock** [reply]
Thank you so much!

I'm new to this and was wondering how to trigger an invitation email to the application. It is okay if it is the "original" sign up page new users see when inviting through the admin dashboard.
